docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.7
+
+# Better movement with [Ctrl + →] and [Ctrl + ←]
+ENV INPUTRC=/etc/inputrc
+
+RUN set -ex \
+  && echo '"\e[1;5C": forward-word' >> /etc/inputrc \
+  && echo '"\e[1;5D": backward-word' >> /etc/inputrc
+
+# Set working directory.
+# Mount your project under the same path.
+WORKDIR /usr/src
+
+# Install useful programs
+RUN apt-get update -y && apt-get install -y \
+      less \
+      git
+
+# Install DVC, supporting all remotes
+RUN pip install "dvc[all]"
+
+# Enable autocompletion scripts for DVC
+RUN wget \
+    -O /etc/bash_completion.d/dvc \
+    https://raw.githubusercontent.com/iterative/dvc/master/scripts/completion/dvc.bash
```

Comment:
Not sure I understand why docker image needs  bash completion. And if it does, why not zsh as well?

Reply:
Usability. We opted to use Bash instead of Zsh. I don't see the point of using both.